### PR TITLE
Add debug command for finding errors.

### PR DESCRIPTION
**Pull request by briantkennedy**
Intent is to make it easier to find inputs that encounter errors while running validator (prints raw input for cut / paste into tests).

### CODE REVIEW

File: cmd/policy-tool/debug/debug.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	"strings"
+
+	"github.com/forseti-security/config-validator/pkg/gcv"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "debug",
+	Short:   "Run the config validator on a set of policies / cai data and print out any info on errors.",
+	Example: `policy-tool debug --policies ./forseti-security/policy-library/policies --libs ./forseti-security/policy-library/libs`,
```
Comment (morgante) [inline cmd/policy-tool/debug/debug.go]: Should this include the file arg?

File: cmd/policy-tool/debug/debug.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+func init() {
+	Cmd.Flags().StringSliceVar(&flags.policies, "policies", nil, "Path to one or more policies directories.")
+	Cmd.Flags().StringVar(&flags.libs, "libs", "", "Path to the libs directory.")
+	Cmd.Flags().StringSliceVar(&flags.files, "file", nil, "Files to process.")
+	if err := Cmd.MarkFlagRequired("policies"); err != nil {
+		panic(err)
+	}
+}
+
+func lintCmd(cmd *cobra.Command, args []string) error {
```
Comment (morgante) [inline cmd/policy-tool/debug/debug.go]:
```suggestion
func debugCmd(cmd *cobra.Command, args []string) error {
```

File: cmd/policy-tool/debug/debug.go (hunk 1 above)
Comment (briantkennedy) [reply cmd/policy-tool/debug/debug.go]: added file flag

File: cmd/policy-tool/debug/debug.go (hunk 2 above)
Comment (briantkennedy) [reply cmd/policy-tool/debug/debug.go]: oof, the embarrassment of not completely fixing a cut and paste job, fixed.

**Pull request merged**